megazords/ios-rust/MozillaTestServices/MozillaTestServices.xcodeproj/project.pbxproj

```diff
@@ -58,6 +58,7 @@
 		1BF50F1927B1E19500A9C8A5 /* FxAccountManagerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1BBAC54127AE065300DAFEF2 /* FxAccountManagerTests.swift */; };
 		1BF50F1A27B1E19800A9C8A5 /* FxAccountMocks.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1BBAC53C27AE065300DAFEF2 /* FxAccountMocks.swift */; };
 		1BFC469827C99F250034E0A5 /* Metrics.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1BFC469627C99F250034E0A5 /* Metrics.swift */; };
+		3963A5862919A541001ED4C3 /* JSONUtil.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3963A5852919A541001ED4C3 /* JSONUtil.swift */; };
```

Comment:
So I've not worked on iOS projects within git repos before -- what kind of changes should we expect to be made to the files like this that we should be committing back to the repo?

Reply:
This is just what adding a new swift file to the Xcode project looks like. Xcode project files are just a bit gross to read by a huma, but, unfortunately we need to commit it to the repo. We don't edit this manually so I don't think it is as important to review manually.